frontend/web/styles/3rdParty/_hljs.scss

```
@@ -1,6 +1,6 @@
 @import '../mixins/custom-scrollbar';
 .custom-scroll {
-  @include customScroll()
+  @include customScroll();
```

Stylelint (17.14.1)

[error] Unexpected parentheses in argumentless mixin "customScroll" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)